src/components/ReviewConfirm/IncorporationDateTime.vue

```diff
@@ -73,15 +73,26 @@
                 id="am-pm-selector"
                 :items="timePeriod"
                 v-model="selectPeriod"
-                :disabled="!isFutureEffective"
+                :disabled="!isFutureEffective || !dateText"
                 filled
               ></v-select>
             </v-col>
             <v-col cols="12" sm="6" md="2" class="label-col">
               <span class="time-zone-label" :class="{ 'disabled': !isFutureEffective }">Pacific Time</span>
             </v-col>
           </v-row>
-        </div>
+          <v-row>
```

Comment:
I don't think this is the right way to do validations and errors.
Have you looked at Vuelidate usage in https://github.com/bcgov/sbc-common-components/blob/master/vue/sbc-common-components/src/components/BaseAddress.vue ?
There's also built-in Vuetify validation (using rules -- which can be dynamic, so you can have different messages).

Reply:
For sure, this little section had to be custom because it wasn't tied to an individual input field, instead a group. When we use the built in validations, it is tied to its component, where i wanted this message to appear under all the selectors as a group. Maybe a better way but this wasn't overly complicated either :)